randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -365,102 +577,182 @@ Extra - asset_id: collision_camera_009
+> Door to collision_camera_009 (G) (Missile Lock); Heals? False
+  * Power Beam Door to collision_camera_009 (G)/Door to Super Missile Room (Missile Lock)
   * Extra - actor_name: doorpowerpower_007
   * Extra - actor_def: actordef:actors/props/doorpowerpower/charclasses/doorpowerpower.bmsad
   * Extra - left_shield_entity: {EMPTY}
   * Extra - left_shield_def: None
   * Extra - right_shield_entity: Root:pScenario:rEntitiesLayer:dctSublayers:default:dctActors:doorshieldmissile_000
   * Extra - right_shield_def: actordef:actors/props/doorshieldmissile/charclasses/doorshieldmissile.bmsad
+  > Pickup (itemsphere_supermissile)
+      Trivial
 
 > Pickup (itemsphere_supermissile); Heals? False
   * Pickup 100; Major Location? True
   * Extra - actor_name: itemsphere_supermissile
   * Extra - actor_def: actordef:actors/items/itemsphere_supermissile/charclasses/itemsphere_supermissile.bmsad
+  > Door to collision_camera_009 (G) (Missile Lock)
+      Trivial
 
 > Start Point; Heals? False; Spawn Point
   * Extra - start_point_actor_name: SP_Checkpoint_SuperMissiles
   * Extra - start_point_actor_def: actordef:actors/logic/startpoint/charclasses/startpoint.bmsad
+  > Door to collision_camera_009 (G) (Missile Lock)
+      Trivial
+
+> Dock to collision_camera_009 (G); Heals? False
+  * Morph Ball Tunnel to collision_camera_009 (G)/Dock to Super Missile Room
+  > Door to collision_camera_009 (G) (Super Lock)
+      Morph Ball
 
 ----------------
 collision_camera_011 (G)
 Extra - total_boundings: {'x1': -8000.0, 'x2': -4900.0, 'y1': -1200.0, 'y2': 800.0}
 Extra - polygon: [[-4900.0, 800.0], [-8000.0, 800.0], [-8000.0, -1200.0], [-4900.0, -1200.0]]
 Extra - asset_id: collision_camera_011
-> Dock to collision_camera_004 (G); Heals? False
-  * Not Determined to collision_camera_004 (G)/Dock to collision_camera_011 (G)
+> Door to Blue EMMI Introduction; Heals? False
+  * Power Beam Door to Blue EMMI Introduction/Door to collision_camera_011 (G)
   * Extra - actor_name: doorpowerpower_002
   * Extra - actor_def: actordef:actors/props/doorpowerpower/charclasses/doorpowerpower.bmsad
   * Extra - left_shield_entity: Root:pScenario:rEntitiesLayer:dctSublayers:default:dctActors:door_shield_plasma_001
   * Extra - left_shield_def: actordef:actors/props/door_shield_plasma/charclasses/door_shield_plasma.bmsad
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
+  > Dock to collision_camera_012 (G) (dooremmy_002)
+      Trivial
 
 > Dock to collision_camera_012 (G) (dooremmy_002); Heals? False
   * EMMI Door to collision_camera_012 (G)/Dock to collision_camera_011 (G) (dooremmy_002)
   * Extra - actor_name: dooremmy_002
   * Extra - actor_def: actordef:actors/props/dooremmy/charclasses/dooremmy.bmsad
+  > Door to Blue EMMI Introduction
+      Trivial
+  > Dock to collision_camera_012 (G) (dooremmy_003)
+      Any of the following:
+          Grapple Beam or Use Spin Boost
+          Gravity Suit and Simple IBJ
```

Comment:
afaik connections that use spin boost should always explicitly specify space jump as well

Reply:
"Use Spin Boost" is a template that is satisfied with either spin boost or space jump